packages/taquito-http-utils/src/taquito-http-utils.ts

```diff
@@ -74,60 +67,63 @@ export class HttpBackend {
   /**
    *
    * @param options contains options to be passed for the HTTP request (url, method and timeout)
-   * @throws {@link HttpRequestFailed} | {@link HttpResponseError}
+   * @throws {@link HttpRequestFailed} | {@link HttpResponseError} | {@link HttpTimeoutError}
    */
   async createRequest<T>(
-    { url, method, timeout = this.timeout, query, headers = {}, json = true }: HttpRequestOptions,
+    { url, method, query, timeout = this.timeout, headers = {}, json = true }: HttpRequestOptions,
```

Comment:
createRequest is an exported function the order of params changes might create breaking change for user. Worth thinking about what value does the order change brings and does it out weight creating breaking change?

Reply:
I'm pretty sure this destructure does not hold sequence, hence no breaking change. But I can revert the sequence back for aesthetics

Reply:
that's great realized it's an object the order doesn't matter will resolve